rust-src/concordium_base/src/ecvrf/proof.rs

```diff
@@ -29,24 +29,26 @@ pub fn hash_points(pts: &[CompressedEdwardsY]) -> Scalar {
 /// a given public key.
 pub struct Proof(pub EdwardsPoint, pub Scalar, pub Scalar);
 
-/// Implements step 8 of <https://tools.ietf.org/id/draft-irtf-cfrg-vrf-07.html#rfc.section.5.1>
```

Comment:
As we discussed, we may just correct the link without transitioning to the stable version!

Reply:
Done; we use the correct URL pointing to the draft version now.